src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write code here
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+
+const MAX_SIZE = 10;
+const MIN_SIZE = 2;
+
+function updateButtons() {
+  const rowCount = table.rows.length;
+  const colCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row and one cell initially to avoid errors when accessing `table.rows[0].cells.length`. If the table is empty, this line will throw an error.